### PR TITLE
Change expander glyph size

### PR DESCRIPTION
**Pull request by StephenLPeters**
Update the expander glyphs to match our designs.

### CODE REVIEW

File: dev/Expander/Expander_themeresources.xaml [hunk 1]
```diff
@@ -338,7 +339,8 @@
                             BorderThickness="{ThemeResource ExpanderChevronBorderThickness}"
                             Background="{ThemeResource ExpanderChevronBackground}">
                             <local:AnimatedIcon
-                                Margin="7"
+                                Width="{StaticResource ExpanderChevronGlyphSize}"
```
Comment (chingucoding) [inline dev/Expander/Expander_themeresources.xaml:342]: Just to check, is removing the margin in favor using width and height correct here?

Comment (StephenLPeters) [reply dev/Expander/Expander_themeresources.xaml:342]: I think so, we could have it scale with its parent size by using a margin to limit the size available or set the size explicitly. I think the explicit size makes mores sense here.

**Review by ranjeshj — APPROVED** [review]

**Comment by ranjeshj** [comment]
@StephenLPeters can you post a picture ?

**Review by chingucoding — APPROVED** [review]

**Pull request merged**